lib/puppet_x/msutter/templates/invoke_dsc_resource.erb

```diff
@@ -42,12 +42,13 @@ $invokeParams = @{
 <% dsc_parameters.each do |p| -%>
     <%= p.name.to_s.gsub(/^dsc_/,'') %> = <%= format_dsc_value(p.value) %>
 <% end -%>
-  }
-<% if resource[:dscmeta_module_name].casecmp('PSDesiredStateConfiguration') == 0 -%>
+  }<% if resource.parameters[:dscmeta_module_version] %>
   ModuleName = @{
-    <%= "ModuleName=\"#{resource[:dscmeta_module_name]}\"" %>
-    <%= "RequiredVersion=\"#{resource[:dscmeta_module_version]}\"" if resource.parameters[:dscmeta_module_version] %>
+    ModuleName      = <%= "\"#{resource[:dscmeta_module_name]}\"" %>
+    RequiredVersion = <%= "\"#{resource[:dscmeta_module_version]}\"" %>
   }
+<% else %>
+  ModuleName = <%= "\"#{resource[:dscmeta_module_name]}\"" %>
 <% end -%>
```

Comment:
Out of curiosity, why do we need and if else then if we can cover that line in the inline if statement? Is it the extra empty line in ModuleName?

e.g.

```powershell
ModuleName @{
   ModuleName = "somename"

}
```

Does it error when it's an array with only one element? That's what I'm guessing...